app/models/like.rb

```diff
@@ -1,4 +1,9 @@
 class Like < ApplicationRecord
-  belongs_to :author
-  belongs_to :post
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+  after_create :update_like_count
+end
+
+def update_like_count
+  post.increment!(:likes_counter)
```

Comment:
- It would be nice to use one of the Rails Active Callbacks - `after_save` to call the #update_like_counter method every time a new post has been saved to the database. 😃